src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/AncmV2.wixproj

```diff
@@ -45,6 +45,7 @@
     <ProjectReference Include="..\CustomAction\aspnetcoreCA.vcxproj">
       <Name>aspnetcoreCA</Name>
       <SetPlatform Condition="'$(Platform)' == 'x86'">Platform=Win32</SetPlatform>
+      <SetPlatform Condition="'$(Platform)' == 'arm64'">Platform=ARM64</SetPlatform>
```

Comment:
Curious, why isn't this needed for x64?

Reply:
Just guessing that x64 is the default perhaps, will leave this unresolved to follow up on this later